### PR TITLE
CBAPI 1.6 - Devices v6, Alerts v6

### PR DESCRIPTION
**Pull request by abowersox-cb**
Also move deviceInfo and create a new REST API object at the PSC level.

_(This pull request will not be merged yet! I just wanted to create something that will submit the changes I'm making for general comment.)_

### CODE REVIEW

File: src/cbapi/psc/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Exported public API for the Cb PSC API
+
+from __future__ import absolute_import
+
+from .rest_api import CbPSCBaseAPI
+from cbapi.psc.models import Device as DeviceV6
```
Comment (avanbrunt-cb) [inline src/cbapi/psc/__init__.py]: Hmm. I am hesitant to call this DeviceV6 as we don't want to have to maintain this name along with any other names in the future. Do we run into a conflict calling this Device?

File: src/cbapi/psc/models.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            except:
+                pass
+
+        self._dirty_attributes = {}
+        if refresh_required:
+            self.refresh()
+        return self._model_unique_id
+
+
+class Device(PSCMutableModel):
```
Comment (avanbrunt-cb) [inline src/cbapi/psc/models.py]: Is this just a place holder for the new device services?

Comment (abowersox-cb) [reply src/cbapi/psc/models.py]: Yes. It's copied from the existing Device object, as you said I should do.

File: src/cbapi/psc/models/deviceInfo.yaml [hunk 3]
```diff
@@ -0,0 +1,221 @@
+type: object
```
Comment (avanbrunt-cb) [inline src/cbapi/psc/models/deviceInfo.yaml]: This is the old device schema and I would recommend just device.yaml

File: src/cbapi/psc/__init__.py (hunk 1 above)
Comment (abowersox-cb) [reply src/cbapi/psc/__init__.py]: We might, but I don't know. I can drop the "as DeviceV6" from that import and see if anything barfs at it.

Comment (abowersox-cb) [reply src/cbapi/psc/__init__.py]: In any event, it'll eventually be called Device when the other Device goes away.

Comment (avanbrunt-cb) [reply src/cbapi/psc/__init__.py]: Looks like the pipeline is still failing

File: src/cbapi/psc/models/deviceInfo.yaml (hunk 3 above)
Comment (abowersox-cb) [reply src/cbapi/psc/models/deviceInfo.yaml]: I just copied it, I'm not changing it for now.

File: src/cbapi/psc/__init__.py (hunk 1 above)
Comment (avanbrunt-cb) [reply src/cbapi/psc/__init__.py]: Right. But that is not deactivating for 6months so lets not have to maintain DeviceV6

**Comment by abowersox-cb** [comment]
Re: build pipeline...I could have just said that I could care less about it right now, because this is nowhere near ready for anything. However, it looks like I messed up some imports, so I'm trying again.

File: src/cbapi/psc/__init__.py (hunk 1 above)
Comment (abowersox-cb) [reply src/cbapi/psc/__init__.py]: It got taken out in the midst of the flurry of changes to get the checks working right. Which they are now. Thank $DEITY. :D

File: src/cbapi/models.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 import sys
```
Comment (avanbrunt-cb) [inline src/cbapi/models.py]: We need to remove this import

Comment (abowersox-cb) [reply src/cbapi/models.py]: Removed.

**Pull request merged**